lib/shared/store.js

```diff
@@ -167,20 +163,20 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
       }
 
       // Convert object instances to plain objects
-      action.data = JSON.parse(JSON.stringify(action.data))
+      const drives = JSON.parse(JSON.stringify(action.data))
```

Comment:
This might be another stupid question, but is there a more efficient way of doing this than transitioning to and from JSON?

Reply:
@Shou why is this object being copied again? I remember there was some reason for it, but since it's not in the comment...

Reply:
We tried using `Object.assign` once but it didn't fulfil the job completely – @jviotti knows the specifics. Maybe we can combine `Object.assign` with a function that takes care of the edge-case you found @jviotti? I imagine it'll be a lot faster than JSON conversion. We can do this in another PR though!

Reply:
I don't remember the details anymore, sorry :(

Reply:
Yes, good point, let's figure that out in a different PR